### DISCUSSION TITLE
Combine an audio and a text file as one single file

### DISCUSSION THREAD

**Discussion by ManimMadhav** [katspaugh/wavesurfer.js#3611] [Q&A]
Is there a way to combine an audio file (either ```.mp3``` or ```.wav```) and store some text information in a ```.txt``` file and save it as one single file (in which format such a file would be, I am not sure).

What I mean is, say I have an audio file in which I want to make regions. Those regions are stored in a text file which contains 3 values, i.e ```start``` and ```end``` times of regions in the audio along with a ```label``` for the region. I want to store the information in the text file saved with the audio in a manner that when a user loads an audio file using ```wavesurfer.load()```, the regions get created automatically. How can i do so? Is this even possible?

**Comment by katspaugh** [comment]
Wavesurfer doesn’t do this by default but you can save your regions as a JSON and re-create them on load.

**Reply by ManimMadhav** [reply]
hey @katspaugh, thanks for your response. 
I have tried and tested this way of storing regions in a JSON file and then imported it. the issue with this is that it gets tougher to manage when the number of audio files a user uses increases. Hence, is there a way i can store this information in the audio file (for eg. in it's ID3 metadata container) itself?

**Reply by katspaugh** [reply]
I don't know. If you find a way to do it, please share here.

**Reply by ManimMadhav** [reply]
okay sure. will update if i find anything useful. thanks for your time!
